system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -472,6 +507,179 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+// Ethereum Bridge
+parameter_types! {
+	pub storage EthereumGatewayAddress: H160 = H160(hex_literal::hex!("EDa338E4dC46038493b885327842fD3E301CaB39"));
+}
+
+parameter_types! {
+	pub const CreateAssetCall: [u8;2] = [53, 0];
+	pub const CreateAssetDeposit: u128 = (UNITS / 10) + EXISTENTIAL_DEPOSIT;
```

Comment:
Let's also have test for that. Things are changing over time and we'll definitely see issue with all those hardcoded values after they'll be deployed on-chain. Better to detect that by tests :)

Reply:
I also suggest maybe to reuse our crates in this repo (`bp-asset-hub-*` and `bp-bridge-hub-*`) as a proper place to define constants that need to be referenced by both AH and BH. You could make a `mod snowbridge` there for the sake of cleanliness

Reply:
@vgeddes would it make sense for us to test the `CreateAssetDeposit` value? It is used in our fee calculation https://github.com/paritytech/polkadot-sdk/blob/master/bridges/snowbridge/primitives/router/src/inbound/mod.rs#L167

Reply:
Maybe we need to reopen https://github.com/Snowfork/snowbridge/pull/1112?